firefox_puppeteer/__init__.py

```diff
@@ -79,6 +79,14 @@ def security(self):
         See the :class:`~api.security.Security` reference.
         """
 
+    @use_class_as_property('api.software_update.SoftwareUpdate')
+    def software_update(self):
+        """
+        Provides support for an easy access to the update process.
+
+        See the :class:`~api.software_update.SoftwareUpdate` reference.
+        """
```

Comment:
Please make sure to add the appropriate sphinx files so that the module gets documented.